android/build.gradle

```diff
@@ -28,23 +30,11 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${project.ext.kotlinPluginVersion}"
         classpath("com.android.tools.build:gradle")
         classpath("com.facebook.react:react-native-gradle-plugin")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin")
     }
 }
 
-subprojects {
-    afterEvaluate {
```

Comment:
Had to get rid of this code block because it was no longer supported.

However adding this as a `gradle` task in `android/app/build.gradle` will get rid of at-least the following patches : 
- `patchKeycardPhase`
- `patchCameraKitPhase`
- `patchShakePhase`